shap/explainers/tree.py

```diff
@@ -81,7 +84,7 @@ def __init__(self, model, **kwargs):
         else:
             raise Exception("Model type not yet supported by TreeExplainer: " + str(type(model)))
 
-    def shap_values(self, X, **kwargs):
+    def shap_values(self, X, tree_limit=-1, **kwargs):
```

Comment:
Great to have this argument. It should probably apply to all the model types though not just XGBoost and LightGBM